### PR TITLE
Update agile-process.md

### PR DESCRIPTION
**Pull request by petermarks-gsa**
Following up on standup today, @tadhg-ohiggins mentioned it wasn't clear who would move issues to done. I'm proposing the following process depending on type of issue. In my experience, it's almost always a good idea to have someone else review work to ensure it clearly meets the acceptance criteria. As of right now, I'm putting the PM (myself) as an accountable reviewer as there's no QA team, but happy to re-evaluate that at some point.

### CODE REVIEW

**Review by jhansen-flexion — APPROVED** [review]

**Review by jhansen-flexion — APPROVED** [review]
thank you!

File: docs/agile-process.md [hunk 1]
```diff
@@ -17,7 +17,17 @@ There is a [high-level roadmap](https://app.mural.co/t/gsa6/m/gsa6/1646336450605
 
 - **In review** - There is code review or QA happening on this feature
 
-- **Done (Sprint X)** - At the time of writing, "done" for a task means it passed code review and was merged/deployed. "Done" for a story means it's been deployed and has had its acceptance criteria created. Sprint-specific "done" columns are used to track exactly what was completed in a current sprint for velocity / reporting purposes.
+- **Done (Sprint X)** - The path to "Done" varies by issue type:
```
Comment (Jkrzy) [inline docs/agile-process.md:20]: ❤️

File: docs/agile-process.md [hunk 2]
```diff
@@ -17,7 +17,17 @@ There is a [high-level roadmap](https://app.mural.co/t/gsa6/m/gsa6/1646336450605
...
+
+    - **User Story** - Acceptance criteria for a task is verified by PM or QA lead on staging, then moved to done. 
+
+    - **Engineering task** - Acceptance criteria is verified by an engineer on staging, then moved to done.
```
Comment (Jkrzy) [inline docs/agile-process.md:24]: We only have a single deployed space at the moment, proposing a change here to reflect that.


```suggestion
    - **User Story** - Acceptance criteria for a task is verified by PM or QA lead on the development instance, then moved to done. 

    - **Engineering task** - Acceptance criteria is verified by an engineer on the development instance, then moved to done. 
```

**Review by Jkrzy — APPROVED** [review]
Thanks @petermarks-gsa, this extra detail is helpful!

**Pull request merged**